### ISSUE TITLE
Any network-level factors that affect detection performance?

### ISSUE THREAD

**Issue by Cli98** [zylo117/Yet-Another-EfficientDet-Pytorch#337]
Hi @zylo117 ,

Hopefully you are doing well. I'm currently working on this repo and try to improve the performance of my customized dataset. However, I'm not able to improve too much. 

Just want to ask, aside from some commonly-known factors (learning rate, batch size, epochs, choice of d, suitable anchors), are there any other possible factors may be possible to improve detection performance? 

Thank you for any hints you can share.

**Comment by zylo117** [comment]
image size

**Comment by Cli98** [comment]
> image size

@zylo117 Emmm... In training phase or test phase?

**Comment by zylo117** [comment]
> > image size
> 
> @zylo117 Emmm... In training phase or test phase?

both
